internal/app/service/certificate_service.go

```diff
@@ -5,13 +5,15 @@ import (
 
 	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/dto"
 	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/errors"
+	"github.com/joshsoftware/profile_builder_backend_go/internal/pkg/helpers"
 	"github.com/joshsoftware/profile_builder_backend_go/internal/repository"
 	"go.uber.org/zap"
 )
 
 // CertificateService represents a set of methods for accessing the certificates.
 type CertificateService interface {
 	CreateCertificate(ctx context.Context, cDetail dto.CreateCertificateRequest) (profileID int, err error)
+	GetCertificates(ctx context.Context, profileID string) (value []dto.CertificateResponse, err error)
```

Comment:
try to keep same type vars for `profileID`

Reply:
kk